guide/07-working-with-bigdata/01-introduction.ipynb

```diff
@@ -180,6 +180,13 @@
    "source": [
```

Comment:
<p>GeoAnalytics Server works with your vector (points, lines, and polygons) and tabular data and can read directly from CSV files, .txt files, shapefiles, and big data sources, such as cloud storage, HDFS, and Hive. GeoAnalytics Server also works with your existing GIS data, using feature layers as input.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>